docs/src/main/asciidoc/transaction.adoc

```diff
@@ -362,16 +362,21 @@ NOTE: The `event` object represents the transaction ID, and defines `toString()`
 TIP: In listener methods, you can access more information about the transaction in progress by accessing the `TransactionManager`,
 which is a CDI bean and can be ``@Inject``ed.
 
-== Configuring transaction log to be stored in a DataSource
+[[jdbcstore]]
+== Configure the transaction log to be stored in a dataSource
 
-The Narayana project has the capability to store the transaction logs into a JDBC Datasource; this should be our recommendation for users needing transaction recovery capabilities, especially when running in volatile containers.
+JDBCStore is a class in the Narayana transaction management library that facilitates the storage of transaction logs in a database by using JDBC.
 
-To enable this capability, you need to set `quarkus.transaction-manager.object-store.type` to `jdbc` explicitly. Also, you can specify a datasource name to be used for the transaction log storage by setting `quarkus.transaction-manager.object-store.datasource`. It will use the default datasource configuration if not specified.
+JDBCStore is our recommended approach for users needing transaction recovery capabilities, especially when running in volatile containers.
```

Comment:
Where does the recommendation come from? I can understand why it might be easier to use in a cloud environment but the file or journal stores are normally recommended. Even volatile containers are normally configured to use persistent volumes.

The reason I've picked on this statement is that the JDBCStore is a lot slower than the journal or file system and  performance of transactions is one of the primary concerns with using transactions.

Reply:
I worked with the original text where the recommendation was already applied. Let me draft a second version of this whole section so that I can help @zhfeng with this a bit.

Reply:
@mmusgrov Whats are the benefits of this feature then, please?

Reply:
> @mmusgrov Whats are the benefits of this feature then, please?

Yoann suggested some nice text earlier that you can use here (`Database storage of transaction logs generally performs worse than journal of filesystem storage, but may be necessary in cloud environment lacking persistent storage (for example if your application container cannot use persistent volumes).`